__tests__/app.test.js

```diff
@@ -74,7 +75,86 @@ describe("app.js", () => {
           });
       });
     });
+    describe("PATCH /api/articles/:article_id - in/decrement article vote by given {inc_votes} property value", () => {
+      test("200: Correctly updates vote property, returns the article", () => {
+        return request(app)
+          .patch("/api/articles/1")
+          .send({ inc_votes: 5 })
+          .then(201)
+          .then((response) => {
+            const { body } = response;
+            expect(body.votes).toBe(105);
+          });
+      });
+      test("200: Correctly updates vote property, returns the article", () => {
+        return request(app)
+          .patch("/api/articles/1")
+          .send({ inc_votes: -50 })
+          .expect(200)
+          .then((response) => {
+            const { body } = response;
+            expect(body.votes).toBe(50);
+          });
+      });
+      test("304 - Not modified: Doesn't allow votes to go into negatives", () => {
```

Comment:
This is a perfectly fine test but just be aware that sites like reddit, you can downvote into negatives. It is entirely up to you how you design your server